romancal/conftest.py

```diff
@@ -84,3 +76,32 @@ def pytest_runtest_logstart(self, nodeid, location):
             yield
             if self.desc:
                 self.terminal_reporter.write(f"\n{self.desc} ")
+
+
+@pytest.hookimpl(trylast=True)
+def pytest_configure(config):
+    terminal_reporter = config.pluginmanager.getplugin("terminalreporter")
+    config.pluginmanager.register(
+        TestDescriptionPlugin(terminal_reporter), "testdescription"
+    )
+
+    # ADD default skip markers
+    config.addinivalue_line(
+        "markers", "webbpsf: mark test as requiring webbpsf data to run"
+    )
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--webbpsf", action="store_true", default=False, help="run webbpsf tests"
```

Comment:
Technically, we could set `default` to be true if users happen to have the `WEBBPSF_PATH` environment variable set.